keyboards/planck/keymaps/vennjadin/keymap.c

```diff
@@ -0,0 +1,230 @@
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+ * ,-----------------------------------------------------------------------------------.
+ * | Esc  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Tab  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   Ø  |  Æ   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   Å  |Ent|shift|
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | 1pass| Ctrl | Alt  | GUI  |Lower |    Space    |Raise | GUI  | Alt  |Mute  |@     |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_QWERTY] = LAYOUT_planck_grid(
+    KC_ESC,      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
+    KC_TAB,      KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    NO_OSTR, NO_AE,
+    KC_LSFT,     KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  NO_ARNG, KC_SFTENT,
+    LSG(KC_SPC), KC_LCTL, KC_LALT, KC_LGUI, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_RGUI, KC_RALT, KC_MUTE,   NO_QUOT
+),
+
+/* Lower
+ * ,-----------------------------------------------------------------------------------.
+ * |      |   +  |   7  |   8  |   9  |   ,  |   %  | [    |  ]   |  =   |   _  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  -   |  4   |  5   |  6   |  .   |  /   |   (  |  )   | <    |   >  |  '   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  0   |  1   |  2   |  3   |  *   |  \   |   {  |  }   |  #   |   $  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |enter |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_LOWER] = LAYOUT_planck_grid(
+    _______, NO_PLUS, KC_7,    KC_8,    KC_9,    KC_COMM, KC_PERC,    NO_LBRC,    NO_RBRC,    NO_EQL,  NO_UNDS, _______,
+    _______, NO_MINS, KC_4,    KC_5,    KC_6,    KC_DOT,  NO_SLSH,    NO_LPRN,    NO_RPRN,    NO_LABK, NO_RABK, NO_PIPE,
+    _______, KC_0,    KC_1,    KC_2,    KC_3,    NO_ASTR, A(NO_SLSH), A(NO_LPRN), A(NO_RPRN), NO_HASH, NO_CURR, _______,
+    KC_ENT,  _______, _______, _______, _______, _______, _______,    _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
+),
+
+/* Raise
+ * ,-----------------------------------------------------------------------------------.
+ * |      |  ~   |  ¨   |   `  |  ´   |   ^  |      | BrD  |  up  |  BrU |      |      |
```

Comment:
```suggestion
 * |      |  ~   |  ¨   |   `  |  \   |   ^  |      | BrD  |  up  |  BrU |      |      |
```
The code says it is `NO_BSLS`